src/hooks/use-wp-list.ts

```diff
@@ -0,0 +1,60 @@
+import * as Sentry from '@sentry/electron/renderer';
+import { useState, useEffect } from 'react';
+import { getIpcApi } from 'src/lib/get-ipc-api';
+
+export type WpListType = 'plugins' | 'themes';
+export type WpListEntry = { name: string; type: 'file' | 'folder' };
+export type WpListResult = { items: WpListEntry[]; isLoading: boolean; error: Error | null };
+
+export function useWpList(
```

Comment:
We can consider renaming this hook to something more descriptive. Maybe `useWpContentFolders`.